data_structures/bloom_filters/bloom_filter/main.c

```diff
@@ -0,0 +1,21 @@
+#include "./bloom_filter.h"
+#include <sys/resource.h>
+
+int main(void) {
+    struct BloomFilter * bf = construct_bloomFilter(100, .001);
+    int aP;
+    for(int i = 0; i < 10; ++i) {
+        
+        int * p = malloc(sizeof *p);
+        *p = i;
+        aP = *p;
+        bloom_insert(bf, p, sizeof *p);
+        free(p);
+    }
+    bloom_printStats(bf);
```

Comment:
I don't really understand why you allocate and free that pointer every time, the same goes for that aP = *p; Maybe I'm being dumb, but in case I am not, here's the correction
```suggestion
    int * p = malloc(sizeof *p);
    for(int i = 0; i < 10; ++i) {
        *p = i;
        bloom_insert(bf, p, sizeof *p);
    }
    free(p);
    int aP = 9;
    bloom_printStats(bf);
```